framework/OpenMod.Core/Localization/NullStringLocalizer.cs

```diff
@@ -24,10 +24,6 @@ public IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures)
             return Enumerable.Empty<LocalizedString>();
         }
 
-        public IStringLocalizer WithCulture(CultureInfo culture)
-        {
-            return this;
-        }
```

Comment:
Maybe instead of removing, mark it as obsolete and just return IStringLocalizer

Reply:
done

Reply:
It should be removed, it's now unused in OpenMod. Obsolete will not work here, due to using `IStringLocalization` interface.

Reply:
Done